src/content/docs/tutorial-service-level-mgmt/intro-to-slm.mdx

```diff
@@ -0,0 +1,156 @@
+---
+
+import omSlTheWholeIsGreater from 'images/om-sl_diagram_theWholeIsGreater.webp'
+
+import apmOverviewScreen from 'images/apm_screenshot-full_overview-screen.webp'
+
+import apmServiceBoundary from 'images/apm_screenshot-crop_service-boundary.webp'
+
+import apmLatencyBaseline from 'images/apm_screenshot-full_latency-baseline.webp'
+
+import apmSLStartFromApm from 'images/apm_screenshot-full_sl-start-from-apm.webp'
+
+import apmAddInputServiceLevel from 'images/apm_screenshot-crop_add-input-service-level.webp'
+
+import apmSyntheticsSetup from 'images/apm_screenshot-crop_synthetics-setup.webp'
+
+import queriesnrqlTransactionListNrql from 'images/queries-nrql_screenshot-full_transaction-list-nrql.webp'
+
+import apmCapabilityFilterClause from 'images/apm_screenshot-crop_capability-filter-clause.webp'
+
+
+Within an organization, communication between stakeholders can often prove more difficult than needed. IT departments usually don't speak in terms that the business-focused parts of an organization can understand, and they in turn don't normally communicate in terms that IT teams find useful. To improve reliability, resolving this language barrier helps to prevent issues. Service level management, or the practice of standardizing data into a universal language that all parties can understand, aids in doing just that.
+
+Service level management best applies to the practice of **Uptime, performance and reliability**, but it also applies to the other practices of **Customer experience**, **Innovation and growth**, and **Operational efficiency** ([learn more about these practices](/docs/new-relic-solutions/observability-maturity/introduction)). Regardless of what area you want to improve, using SLM has two major areas of impact: business outcomes and operational outcomes.
+
+<Tabs stacked>
+	<TabsBar>
+        <TabsBarItem id="1">
+            Business outcome
+        </TabsBarItem>
+        <TabsBarItem id="2">
+            Operational outcome
+        </TabsBarItem>
+    </TabsBar>
+
+    <TabsPages>
+        <TabsPageItem id="1">
+        The required business outcome in terms of reliability focuses on reducing the number of business-impacting incidents, their duration, and the number of people involved.
+
+        1. Reduce the number of business-disrupting incidents.
+        2. Reduce mean time to resolution (MTTR).
+        3. Reduce average number of people engaged (FTEs) per severe incident.
+        </TabsPageItem>
+        <TabsPageItem id="2">
+        The required operational outcome in terms reliability focuses on communicating the health of your digital product. You measure operational success by what percentage of critical product applications you cover with your standard service levels. You can also look at the percentage of policy adoption by your primary stakeholders. You can do both by focusing on what is important to the stakeholders, ensuring simplicity, and proving the effectiveness of service level management.
```

Comment:
This is a little opaque to me, and almost seems more business oriented. MTTR/incident response/incidents in general seem more operational, but this tab seems more in the realm of business. 

* Operational outcome is about communicating health about a product
* You measure your success by how well you can communicate health
* Communicating health is done through standard service levels

There are other methods to judge operational outcomes which can also include:

* Policy adoption by stakeholders
* Keeping [something] simple (keeping communication about the product simple? keeping the product itself? simple?)
* Creating effective service levels

If this is correct, I might simplify and unspool this paragraph. I had to reread it a few times to understand what exactly was being talked about

Reply:
Reworked to be more clear, thank you! Also I changed my mind on the bulleted list and decided to structure it that way after all.